demo/src/main/java/com/example/demo/model/Person.java

```diff
@@ -2,82 +2,39 @@
 
 
 import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.Getter;
+import lombok.Setter;
 
 import javax.persistence.*;
+import javax.validation.Valid;
 import java.time.LocalDate;
 import java.util.List;
 
-
+@Setter
+@Getter
 @Entity
+@Table(name = "demo_person")
 public class Person {
     @Id
-    @Column(name = "person_id")
     @GeneratedValue
+    @Column(name = "demo_person_id")
     private long id;
+    @Column(name = "demo_given_name")
     private String givenName;
+    @Column(name = "demo_family_name")
     private String familyName;
+    @Column(name = "demo_date_of_birth")
     private LocalDate dateOfBirth;
+    @Column(name = "demo_place_of_birth")
     private String placeOfBirth;
 
-    @JoinColumn(name = "person_id")
+    @JoinColumn(name = "person_demo_person_id")
```

Comment:
incorrect column name, you should change it to `demo_person_id`  as that's the name of the Id column